Gaijinizer patch/Map003.txt

```diff
@@ -1503,50 +1503,54 @@ the number of registrations.#MTLed
  本来なら、気力を最大まで強化しないと厳しいが
 \\C[2]今回は百鬼のアイテムのおかげで、初期気力でも使える。
 > CONTEXT: Map003/events/1/pages/2/389/Dialogue/クッション君
-And this one is stupid for consumption vitality.
-Originally, it is tough if you do not strengthen
-the vitality to the maximum. \\C[2]This time, thanks to
-Hyakki's items, you can use the initial vitality
-as well.#MTLed
+Also, the vitality consumption for this one is
+stupid high. Ordinarily, it would be difficult if
+you didn't upgrade vitality to the maximum, but...
+
+\\C[2]This time, thanks to Hyakki's items, it can
+be used even with the base amount of vitality.#Strate
```

Comment:
Feel free to localise this part to something more natural. For example:

``\\C[2]This time, thanks to Hyakki's items, you'll
be able to get away with even your base level of
vitality.``

Just mark it as #StrateLocalization if you make any major changes from the original.

Reply:
Got it, I changed the wording a bit and marked it accordingly.